### PR TITLE
Fix issue 5140

### PR DESCRIPTION
**Pull request by cristinamanum**
Do not pull in the Support page the support requests for the deleted accounts.

### CODE REVIEW

**Comment by joelverhagen** [comment]
Can we get a unit test? I see no reason why support requests shouldn't have at least some unit tests (maybe it already does?).

File: src/NuGetGallery/Areas/Admin/Controllers/SupportRequestController.cs [hunk 1]
```diff
@@ -231,7 +231,7 @@ public ActionResult History(int id)
 
             var skip = (pageNumber - 1) * take;
             var galleryUsername = GetLoggedInUser();
-            var issues = _supportRequestService.GetIssues(assignedTo, reason, issueStatusId, galleryUsername);
+            var issues = _supportRequestService.GetIssues(assignedTo, reason, issueStatusId, galleryUsername).Where(i => i.CreatedBy != null);
```
Comment (chenriksson) [inline src/NuGetGallery/Areas/Admin/Controllers/SupportRequestController.cs]: If we always ignore / filter these out, why not delete a user's support requests when their account is deleted?

Comment (cristinamanum) [reply src/NuGetGallery/Areas/Admin/Controllers/SupportRequestController.cs]: The decision to keep these was for eventual future diagnostics.

Comment (cristinamanum) [reply src/NuGetGallery/Areas/Admin/Controllers/SupportRequestController.cs]: Added tests

**Pull request merged**